packages/graphql/src/translate/create-interface-projection-and-params.ts

```diff
@@ -19,185 +19,51 @@
 
 import type { ResolveTree } from "graphql-parse-resolve-info";
 import { asArray, removeDuplicates } from "../utils/utils";
-import { AUTH_FORBIDDEN_ERROR } from "../constants";
 import type { ConnectionField, Context, GraphQLOptionsArg, InterfaceWhereArg, RelationField } from "../types";
 import filterInterfaceNodes from "../utils/filter-interface-nodes";
 // eslint-disable-next-line import/no-cycle
 import createConnectionAndParams from "./connection/create-connection-and-params";
-import { createAuthAndParams } from "./create-auth-and-params";
+import { createAuthPredicates } from "./create-auth-and-params";
 // eslint-disable-next-line import/no-cycle
 import createProjectionAndParams from "./create-projection-and-params";
-import { getRelationshipDirectionStr } from "../utils/get-relationship-direction";
-import createElementWhereAndParams from "./where/create-element-where-and-params";
+import { getRelationshipDirection } from "../utils/get-relationship-direction";
 import * as CypherBuilder from "./cypher-builder/CypherBuilder";
 import { addSortAndLimitOptionsToClause } from "./projection/subquery/add-sort-and-limit-to-clause";
 import { compileCypherIfExists } from "./cypher-builder/utils/utils";
+import type { Node } from "../classes";
+import { createWherePredicate } from "./where/create-where-predicate";
+import { AUTH_FORBIDDEN_ERROR } from "../constants";
 
-function createInterfaceProjectionAndParams({
+export default function createInterfaceProjectionAndParams({
```

Comment:
Any reason for the default export? It feels to me like we're moving away from this direction in most cases.

Reply:
Oh, it is just because that was what were being exported, and to reduce the scope of the refactor I kept it as such